.gitignore

```diff
@@ -2,6 +2,8 @@
 *.egg-info
 *.pyc
 .idea
+.poetry.lock
```

Comment:
`poetry.lock` need to be shared with others to speed up install and have common deps.